installation error

Dart is installed via brew. Than:
➜  ~  brew install gazebo9 --build-from-source
==> Installing gazebo9 from osrf/simulation
==> Downloading http://gazebosim.org/distributions/gazebo/releases/gazebo-9.4.1.tar.
Already downloaded: /Users/eva/Library/Caches/Homebrew/downloads/824f01976c997ed9a53a785b169bfc453f0f0b5c73a6341dfc9be061d0fee7e2--gazebo-9.4.1.tar.bz2
==> Downloading https://gist.githubusercontent.com/scpeters/9199370/raw/afe595587e38
Already downloaded: /Users/eva/Library/Caches/Homebrew/downloads/12171d5787bf5709495604c1591fd598a9cffc3a055eb509bea588d33019cabc--brew_python_fix.patch
==> Patching
==> Applying brew_python_fix.patch
patching file deps/opende/CMakeLists.txt
==> cmake .. -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DC
==> make install
Last 15 lines from /Users/eva/Library/Logs/Homebrew/gazebo9/02.make:
[ 84%] Automatic MOC for target PressurePlugin
cd /tmp/gazebo9-20181024-50520-1yffoo4/gazebo-9.4.1/build/plugins && /usr/local/Cellar/cmake/3.12.3/bin/cmake -E cmake_autogen /tmp/gazebo9-20181024-50520-1yffoo4/gazebo-9.4.1/build/plugins/CMakeFiles/PressurePlugin_autogen.dir/AutogenInfo.cmake Release
[ 84%] Built target BuoyancyPlugin_autogen
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f plugins/CMakeFiles/ContactPlugin_autogen.dir/build.make plugins/CMakeFiles/ContactPlugin_autogen.dir/depend
cd /tmp/gazebo9-20181024-50520-1yffoo4/gazebo-9.4.1/build && /usr/local/Cellar/cmake/3.12.3/bin/cmake -E cmake_depends "Unix Makefiles" /tmp/gazebo9-20181024-50520-1yffoo4/gazebo-9.4.1 /tmp/gazebo9-20181024-50520-1yffoo4/gazebo-9.4.1/plugins /tmp/gazebo9-20181024-50520-1yffoo4/gazebo-9.4.1/build /tmp/gazebo9-20181024-50520-1yffoo4/gazebo-9.4.1/build/plugins /tmp/gazebo9-20181024-50520-1yffoo4/gazebo-9.4.1/build/plugins/CMakeFiles/ContactPlugin_autogen.dir/DependInfo.cmake --color=
Scanning dependencies of target ContactPlugin_autogen
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f plugins/CMakeFiles/ContactPlugin_autogen.dir/build.make plugins/CMakeFiles/ContactPlugin_autogen.dir/build
[ 84%] Automatic MOC for target ContactPlugin
cd /tmp/gazebo9-20181024-50520-1yffoo4/gazebo-9.4.1/build/plugins && /usr/local/Cellar/cmake/3.12.3/bin/cmake -E cmake_autogen /tmp/gazebo9-20181024-50520-1yffoo4/gazebo-9.4.1/build/plugins/CMakeFiles/ContactPlugin_autogen.dir/AutogenInfo.cmake Release
make[1]: *** [gazebo/gui/CMakeFiles/gazebo_gui.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 84%] Built target GimbalSmall2dPlugin_autogen
[ 84%] Built target PressurePlugin_autogen
[ 84%] Built target ContactPlugin_autogen
make: *** [all] Error 2

--- a/Formula/gazebo7.rb
+++ b/Formula/gazebo7.rb
@@ -3,7 +3,7 @@ class Gazebo7 < Formula
   homepage "http://gazebosim.org"
   url "http://gazebosim.org/distributions/gazebo/releases/gazebo-7.14.0.tar.bz2"
   sha256 "c1e48a2304d8186a3df4fb763262d189a8afe6b1ab82c30e614e336f3cb6034f"
-  revision 2
+  revision 3
 
   head "https://bitbucket.org/osrf/gazebo", :branch => "gazebo7", :using => :hg
 
@@ -14,6 +14,7 @@ class Gazebo7 < Formula
   depends_on "cartr/qt4/qt@4"
   depends_on "doxygen"
   depends_on "freeimage"
+  depends_on "graphviz"
   depends_on "ignition-math2"
   depends_on "ignition-transport"
   depends_on "libtar"

--- a/Formula/gazebo7.rb
+++ b/Formula/gazebo7.rb
@@ -7,6 +7,13 @@ class Gazebo7 < Formula
 
   head "https://bitbucket.org/osrf/gazebo", :branch => "gazebo7", :using => :hg
 
+  bottle do
+    root_url "http://gazebosim.org/distributions/bottles-simulation"
+    sha256 "aa093e0a5a36a6a4f291dcbfcbc04ef04b027f24cbbb7758199a8d8739c3f2c4" => :mojave
+    sha256 "ef7fcbaedff5b3bfd89d8c162633e81549d3eba2b2a68d4fe158688fc034e13e" => :high_sierra
+    sha256 "2b35d23860523b8c5835f599593a5222943aa23fb21d1c458c35781ae7878f86" => :sierra
+  end
+
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build
 

--- a/Formula/gazebo7.rb
+++ b/Formula/gazebo7.rb
@@ -42,6 +42,12 @@ class Gazebo7 < Formula
   conflicts_with "gazebo8", :because => "Differing version of the same formula"
 
   patch do
+    # Fix for compatibility with boost 1.68
+    url "https://bitbucket.org/osrf/gazebo/commits/cc53e4cdd34875dbb99048137f1d27541d12b3d0/raw/"
+    sha256 "f382a668ba2c6a318f3d4b5f616a11ad098973d941cd73a5493f0b1788ae8a42"
+  end
+
+  patch do
     # Fix build when homebrew python is installed
     # keep this patch
     url "https://gist.githubusercontent.com/scpeters/9199370/raw/afe595587e38737c537124a3652db99de026c272/brew_python_fix.patch"

--- a/Formula/gazebo8.rb
+++ b/Formula/gazebo8.rb
@@ -47,6 +47,12 @@ class Gazebo8 < Formula
   conflicts_with "gazebo7", :because => "Differing version of the same formula"
 
   patch do
+    # Fix for compatibility with boost 1.68
+    url "https://bitbucket.org/osrf/gazebo/commits/cc53e4cdd34875dbb99048137f1d27541d12b3d0/raw/"
+    sha256 "f382a668ba2c6a318f3d4b5f616a11ad098973d941cd73a5493f0b1788ae8a42"
+  end
+
+  patch do
     # Fix build when homebrew python is installed
     # keep this patch
     url "https://gist.githubusercontent.com/scpeters/9199370/raw/afe595587e38737c537124a3652db99de026c272/brew_python_fix.patch"

--- a/Formula/gazebo8.rb
+++ b/Formula/gazebo8.rb
@@ -4,16 +4,16 @@ class Gazebo8 < Formula
   url "http://gazebosim.org/distributions/gazebo/releases/gazebo-8.6.0.tar.bz2"
   sha256 "c62aeb4a0a761a3685c7f8caa63e8f8ba588ab2ce5ac7b956c6ddeb1ada7be88"
   revision 3
+  version_scheme 1
+
+  head "https://bitbucket.org/osrf/gazebo", :branch => "default", :using => :hg
+
   bottle do
     root_url "http://gazebosim.org/distributions/bottles-simulation"
     sha256 "91436d3faf5e2c30c17aed48a6567fe3246eb03671f92cb0665e120d0df7ea1b" => :mojave
     sha256 "0c463541a6a0cbaa23f3b601e4c3d19df6a474ce17e4574b31276aa42c23795a" => :high_sierra
     sha256 "aff301f6a8101b29f77bd4763edcfd0657dd45c645823a2afe135b283d05c851" => :sierra
   end
-
-  version_scheme 1
-
-  head "https://bitbucket.org/osrf/gazebo", :branch => "default", :using => :hg
 
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build

--- a/Formula/gazebo8.rb
+++ b/Formula/gazebo8.rb
@@ -4,6 +4,13 @@ class Gazebo8 < Formula
   url "http://gazebosim.org/distributions/gazebo/releases/gazebo-8.6.0.tar.bz2"
   sha256 "c62aeb4a0a761a3685c7f8caa63e8f8ba588ab2ce5ac7b956c6ddeb1ada7be88"
   revision 3
+  bottle do
+    root_url "http://gazebosim.org/distributions/bottles-simulation"
+    sha256 "91436d3faf5e2c30c17aed48a6567fe3246eb03671f92cb0665e120d0df7ea1b" => :mojave
+    sha256 "0c463541a6a0cbaa23f3b601e4c3d19df6a474ce17e4574b31276aa42c23795a" => :high_sierra
+    sha256 "aff301f6a8101b29f77bd4763edcfd0657dd45c645823a2afe135b283d05c851" => :sierra
+  end
+
   version_scheme 1
 
   head "https://bitbucket.org/osrf/gazebo", :branch => "default", :using => :hg

--- a/Formula/gazebo8.rb
+++ b/Formula/gazebo8.rb
@@ -3,7 +3,7 @@ class Gazebo8 < Formula
   homepage "http://gazebosim.org"
   url "http://gazebosim.org/distributions/gazebo/releases/gazebo-8.6.0.tar.bz2"
   sha256 "c62aeb4a0a761a3685c7f8caa63e8f8ba588ab2ce5ac7b956c6ddeb1ada7be88"
-  revision 2
+  revision 3
   version_scheme 1
 
   head "https://bitbucket.org/osrf/gazebo", :branch => "default", :using => :hg
@@ -14,6 +14,7 @@ class Gazebo8 < Formula
   depends_on "boost"
   depends_on "doxygen"
   depends_on "freeimage"
+  depends_on "graphviz"
   depends_on "ignition-math3"
   depends_on "ignition-msgs0"
   depends_on "ignition-transport3"

